docs/user_guide/wflow_model_setup.rst

```diff
@@ -26,84 +26,124 @@ setup methods are exposed. Click on
+    * - :py:meth:`~WflowSbmModel.setup_reservoirs_simple_control`
       - This component generates maps of controlled reservoir areas and outlets as well as parameters with average reservoir area, demand, min and max target storage capacities and discharge capacity values.
-    * - :py:func:`~WflowSbmModel.setup_glaciers`
+      - :py:meth:`~WflowSbmModel.setup_rivers`
+    * - :py:meth:`~WflowSbmModel.setup_glaciers`
       - This component generates maps of glacier areas, area fraction and volume fraction, as well as tables with temperature threshold, melting factor and snow-to-ice conversion fraction.
-    * - :py:func:`~WflowSbmModel.setup_lulcmaps`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_lulcmaps`
       - This component derives several wflow maps based on landuse- landcover (LULC) raster data.
-    * - :py:func:`~WflowSbmModel.setup_lulcmaps_from_vector`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_lulcmaps_from_vector`
       - This component derives several wflow maps based on landuse- landcover (LULC) vector data.
-    * - :py:func:`~WflowSbmModel.setup_lulcmaps_with_paddy`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_lulcmaps_with_paddy`
       - This component derives several wflow maps based on landuse- landcover (LULC) raster data with paddy rice.
-    * - :py:func:`~WflowSbmModel.setup_laimaps`
+      - :py:meth:`~WflowSbmModel.setup_soilmaps`
+    * - :py:meth:`~WflowSbmModel.setup_laimaps`
       - This component sets leaf area index (LAI) climatology maps per month.
-    * - :py:func:`~WflowSbmModel.setup_laimaps_from_lulc_mapping`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_laimaps_from_lulc_mapping`
       - This component sets leaf area index (LAI) climatology maps per month based on landuse mapping.
-    * - :py:func:`~WflowSbmModel.setup_soilmaps`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_soilmaps`
       - This component derives several (layered) soil parameters based on a database with physical soil properties using available point-scale (pedo)transfer functions (PTFs) from literature with upscaling rules to ensure flux matching across scales.
-    * - :py:func:`~WflowSbmModel.setup_ksathorfrac`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_ksathorfrac`
       - This component prepares the saturated hydraulic conductivity horizontal ratio from an existing map.
-    * - :py:func:`~WflowSbmModel.setup_ksatver_vegetation`
+      - :py:meth:`~WflowSbmModel.setup_basemaps`
+    * - :py:meth:`~WflowSbmModel.setup_ksatver_vegetation`
       - This component prepares ksatver from soil and vegetation parameters.
-    * - :py:func:`~WflowSbmModel.setup_rootzoneclim`
+      - :py:meth:`~WflowSbmModel.setup_soilmaps`, :py:meth:`~WflowSbmModel.setup_laimaps`
```

Comment:
We actually have two methods for LAI ``setup_Laimaps`` and ``setup_laimaps_from_lulc_mapping``. Maybe equivalent is enough or shall we be complete? In a way they can always prepare their own map and also use ``setup_grid_from_raster`` or ``setup_areamap`` so we can never really be complete anyhow... The important part is that "LAI" should be present. What do you think? I will be or equivalent for all the concerned functions below.

```suggestion
      - :py:meth:`~WflowSbmModel.setup_soilmaps`, :py:meth:`~WflowSbmModel.setup_laimaps` or equivalent
```